### PR TITLE
update ssh public key, need to use the key value

### PR DESCRIPTION
**Pull request by rayashworth**

### CODE REVIEW

File: ansible/roles/request_ocp_fyre/defaults/main.yml [hunk 1]
```diff
@@ -9,7 +9,8 @@ fyre_ocp_inf_group: "ocp-clusters"
 fyre_inf_node_name: "{{ clusterName | lower }}-inf"
 fyre_site: "svl"
 fyre_ocptype: "ocp"
-ssh_public_key_file: "{{ lookup('file', lookup('env','HOME') + '/.ssh/id_rsa.pub') }}"
```
Comment (websterc87) [inline ansible/roles/request_ocp_fyre/defaults/main.yml:12]: Im curious why did you change from this default?

**Review by websterc87 — CHANGES_REQUESTED** [review]

File: ansible/roles/request_ocp_fyre/defaults/main.yml (hunk 1 above)
Comment (rayashworth) [reply ansible/roles/request_ocp_fyre/defaults/main.yml:12]: oh, as I tested I thought it was just adding the filename into the json, and not the actual content.  Maybe I was misusing the variable.   I did rename to ssh_public_key to make it more about the key value and not the filename.

**Review by websterc87 — APPROVED** [review]
Looks good assuming it has been tested
